custom_components/localtuya/water_heater.py

```diff
@@ -0,0 +1,213 @@
+"""Platform to locally control Tuya-based water heater devices."""
+import logging
+from functools import partial
+
+import voluptuous as vol
+from homeassistant.components.water_heater import (
+    DOMAIN,
+    STATE_ECO,
+    STATE_ELECTRIC,
+    STATE_GAS,
+    STATE_HEAT_PUMP,
+    STATE_HIGH_DEMAND,
+    STATE_PERFORMANCE,
+    WaterHeaterEntity,
+    WaterHeaterEntityFeature
```

Comment:
don't know why it does not find the WaterHeaterEntityFeature name... maybe different HA version?

Reply:
Yes, that might be the case. Would you know which version the workflow checks compare against? I'm using HA 2022.8.5 and this integration works fine with it. I've also checked the HA GitHub and `WaterHeaterEntityFeature` is in both the [2022.8.5 release](https://github.com/home-assistant/core/blob/2022.8.5/homeassistant/components/water_heater/__init__.py#L60) and [dev (default) branch](https://github.com/home-assistant/core/blob/dev/homeassistant/components/water_heater/__init__.py#L60).